sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/AbstractFileCacheSensor.java

```diff
@@ -55,6 +59,11 @@ public void execute(SensorContext context) {
       return;
     }
 
+    if (!runtime.getApiVersion().isGreaterThanOrEqual(Version.create(9, 4))){
```

Comment:
Is there also a `getServerVersion()`? If so, we need to make sure in the tests, that we call the right version check.

Reply:
No, as far as I can tell you cannot ask for the server version.